docs/user-guide/start-a-jitsi-meeting.md

```diff
@@ -4,6 +4,26 @@ title: Start a Jitsi Meeting
 sidebar_label: Start a Jitsi Meeting
 ---
 
-Welcome to the user guide!
+## Desktop Browser
 
-Check back soon!
+1. You need a browser (ideally Chrome from Google, please note our separate information)
```

Comment:
Leave the browser recommendation out please.

Reply:
i will do